pkg/kp/service.go

```diff
@@ -38,3 +39,16 @@ func (c consulStore) RegisterService(manifest pods.Manifest, caPath string) erro
 
 	return c.client.Agent().ServiceRegister(podService)
 }
+
+// Go version of http status check
+func HttpStatusCheck(node string, port int) *http.Response {
+	url := fmt.Sprintf("http://%s:%d/_status", node, port)
+	resp, _ := http.Get(url)
+	return resp
+}
+
+func HttpsStatusCheck(node string, port int) *http.Response {
+	url := fmt.Sprintf("https://%s:%d/_status", node, port)
+	resp, _ := http.Get(url)
```

Comment:
ah ok then. As long as resp isn't nil or have a nil body or something I guess we're OK. I don't know what guarantees the api provides so I had to ask